e2e-tests/recover/seedPhrase.spec.js

```diff
@@ -0,0 +1,44 @@
+const { createTestSubaccount } = require("../utils/account");
+
+describe("Account Recovery Using Seed Phrase", () => {
+    let testAccount;
+    beforeAll(async () => {
+        testAccount = await createTestSubaccount();
+    });
+
+    afterAll(async () => {
+        await browser.close();
+        await testAccount.delete();
+    });
+
+    it("navigates to seed phrase page successfully", async () => {
+        const user = await page.goto("/").then(() => page.getDocument());
+        await user
+            .findByRole("button", {
+                name: "Import Existing Account",
+            })
+            .then((importAccountBtn) => importAccountBtn.click())
+            .then(() => user.findByText("Recover Account"))
+            .then((recoverAccountBtn) => recoverAccountBtn.click());
+
+        expect(page).toMatchURL(/\/recover-seed-phrase$/);
```

Comment:
Do we need to add a 'wait for the navigation to complete' step here? Does Playwright wait for any navigations to finish internally/automatically _before_ it runs this assertion? Usually I would expect this type of assertion to be a race condition where if the browser is slow at navigating, we get an intermittent test failure because `page` hasn't updated yet?